styles/google/catppuccin.user.css

```diff
@@ -0,0 +1,1091 @@
+/* ==UserStyle==
+@name           Google Catppuccin
+@namespace      github.com/catppuccin/userstyles/styles/google
+@homepageURL    https://github.com/catppuccin/userstyles/raw/main/styles/google
+@version        1.0.4
```

Comment:
We don't need to bump the version number prior to release i suggest this is left at `1.0.0`

Reply:
Or even `0.1.0`.

Reply:
aight